index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Sofia"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <div class="logo">
+          <p class="logo1">My Logo</p>
+        </div>
+        <div class="menu"><i class="fa fa-bars"> </i></div>
+      </div>
+
+      <div class="content">
+        <div>
+          <h1 class="heading">I am Saif glad to see you!</h1>
+        </div>
+        <div>
+          <p class="para_text">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me. <br />
+            <br />
+            <span class="extra"
+              ><a href="#" class="connect"> LET'S CONNECT </a></span
+            >
+            <br />
+            <br />
+            <span class="icons">
+              <a href="#"><i class="fa fa-twitter"></i></a>
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+              <a href="#"><i class="fa fa-github"></i></a>
+              <img src="peace.png" alt="" />
+            </span>
+          </p>
+        </div>
+      </div>
+    </div>
+  </body>
```

Comment:
- [ ] Let's take a look back to the original design right below compared to your design, I think there are a few points of improvement, don't you think so?
- [ YOUR DESIGN]
- 
<img width="319" alt="sam" src="https://user-images.githubusercontent.com/79058364/158957053-2106f80f-296c-4142-a8c5-ae47c4dcd9b5.png">

- Please adjust your styling, consider using the same `padding, margin, color, background  images, font-size, font-weight, and positions` as per the Figma design to match this bellow ⏬ 
- 
<img width="275" alt="fimaaaria" src="https://user-images.githubusercontent.com/79058364/158957816-7fc29dc2-066f-4a99-9576-e7bc3b97aca8.png">